aixplain/enums/language.py

```diff
@@ -22,14 +22,59 @@
 """
 
 import logging
-
-from aixplain.utils import config
-from aixplain.utils.request_utils import _request_with_retry
+import os
+import json
+import time
+from datetime import datetime, timedelta
 from enum import Enum
 from urllib.parse import urljoin
+from aixplain.utils import config
+from aixplain.utils.request_utils import _request_with_retry
 
+CACHE_FILE = ".aixplain_cache/languages.json"
+CACHE_DURATION = timedelta(hours=24)
+
+
+def save_to_cache(languages):
```

Comment:
Try to create a generic `save_to_cache` file that could be used across all files.